pkg/crc/cluster/cluster.go

```diff
@@ -41,16 +41,20 @@ const (
 	aggregatorClientCert = "/etc/kubernetes/static-pod-resources/kube-apiserver-certs/configmaps/aggregator-client-ca/ca-bundle.crt"
 )
 
-func CheckCertsValidity(sshRunner *ssh.Runner) (bool, bool, error) {
+func CheckCertsValidity(sshRunner *ssh.Runner) (bool, bool, bool, error) {
```

Comment:
Time for some API doc comments, or to put these 3 bools in a struct ;)

Reply:
I changed it to a map, it is better! :)